CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## Next Release
```

Comment:
There is already a changelog for next release down below

Reply:
Ah, it had a version number already so I didn't see it. Fixed